unskript-ctl/unskript_ctl_notification.py

```diff
@@ -443,13 +463,20 @@ def send_sendgrid_notification(self,
         tar_file_name = f"{target_name}" + '.tar.bz2'
         target_file_name = None
         metadata = None
+        failed_object_character_count = sum((len(str(value)) for value in failed_result.values()))
```

Comment:
This code is replicated above as well

Reply:
Removed redundancy. Moved to common code.